### DISCUSSION TITLE
The conversion cannot be performed.\r\n Text: "\r\n MemberType: System.Double\r\n

### DISCUSSION THREAD

**Discussion by esspijep** [JoshClose/CsvHelper#1238] [Q&A]
Hi,

I am reading from an AWS S3 CSV file and got it to be read as a Stream but I am having this error:

![image](https://user-images.githubusercontent.com/41111784/52982596-ea3de200-3421-11e9-9ae5-4ea87282df5f.png)

This is how I used the code:

```cs
using (var responseStream = response.ResponseStream)
{
	using (var sr = new StreamReader(responseStream))
	{
		var config = new CsvHelper.Configuration.Configuration
		{
			HasHeaderRecord = false
		};

		using (var csv = new CsvReader(sr, config))
		{
			var records = csv.GetRecords<Model>();
			return records;
		}
	}
}
```

I think the issue is the \r\n but if the issue is something else please let me know. Please advise.

**Comment by JoshClose** [comment]
Click on the magnifying glass icon of `Message` and paste the full text in here.

**Reply by salman-mukhtar** [reply]
Hi Josh

I have similar issue like this. I am trying to import a file with no headers. Here is what the code looks like and what exception i am getting.
Hope you can help

EXCEPTION:
{"The conversion cannot be performed.\r\n    Text: ''\r\n    MemberType: System.Int32\r\n    TypeConverter: 'CsvHelper.TypeConversion.Int32Converter'\r\nIReader state:\r\n   ColumnCount: 0\r\n   CurrentIndex: 7\r\n   HeaderRecord:\r\n\r\nIParser state:\r\n   ByteCount: 0\r\n   CharCount: 40\r\n   Row: 1\r\n   RawRow: 1\r\n   Count: 5\r\n   RawRecord:\r\nNL-1474;yso;1511;1782579200;2022-05-01\r\n\r\n"}


CONFIG SETTINGS:
config.Delimiter = ";";
config.MissingFieldFound = null;
config.HasHeaderRecord = false;

**Reply by salman-mukhtar** [reply]
![image](https://user-images.githubusercontent.com/64012802/166891690-82993fc2-cfe7-4bb9-9c90-89489a210aa4.png)

**Comment by Norrch2** [comment]
@esspijep You will probably want to massage the value by getting rid of any non digit characters. From this example, double.TryParse will not be able to conver the values "\r\n" that is in your string.
